packages/terra-slide-panel-manager/package.json

```diff
@@ -29,14 +29,16 @@
   "dependencies": {
     "classnames": "^2.2.5",
     "prop-types": "^15.5.8",
+    "terra-disclosure-manager": "^4.8.0",
```

Comment:
Should the disclosure manager be a peer dep?

Reply:
Not sure, I'll defer to @tbiethman on this

Reply:
It should be, but moving it to peer now won't bepassive. I'll get a PR out to make it a peer dependency (and the one in terra-modal-manager) that we can time with another major release.

Reply:
Basically any component that provides Context-based APIs should be a peer dependency in non-application component packages. I want to warn on this at build-time with a webpack plugin or something; I might work on that over my sabbatical.